docs/calendar.md

```diff
@@ -291,6 +291,28 @@ Temporal.Calendar.from('chinese').dateDifference(
 )  // => P1M2D
 ```
 
+### calendar.**fields**(fields: array<string>) : array<string>
+
+**Parameters:**
+
+- `fields` (array of strings): A list of field names.
+
+**Returns:** a new list of field names.
+
+This method does not need to be called directly except in specialized code.
+It is called indirectly when using the `from()` static methods and `with()` methods of `Temporal.DateTime`, `Temporal.Date`, and `Temporal.YearMonth`.
+
+Custom calendars should override this method if they require more fields with which to denote the date than the standard `era`, `year`, `month`, and `day`.
```

Comment:
Shouldn't the Japanese calendar add `"era"`?  The ISO-8601 calendar has no concept of an era.

Reply:
I'll be addressing that in #901.